jsontableschema/constraints/maxLength.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from .. import exceptions
+
+
+# Module API
+
+def check_maxLength(name, value, max_length):
```

Comment:
even with my comment about camel casing the module, I do acknowledge that I know you are camel casing the function to be consistent with the spec itself, so I'm fine with the camel casing of the functions.